app/code/Magento/Persistent/Test/Unit/Helper/SessionTest.php

```diff
@@ -0,0 +1,187 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Persistent\Test\Unit\Helper;
+
+use Magento\Persistent\Helper\Session as SessionHelper;
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Checkout\Model\Session as CheckoutSession;
+use Magento\Persistent\Helper\Data as DataHelper;
+use Magento\Persistent\Model\SessionFactory;
+use Magento\Persistent\Model\Session;
+
+/**
+ * Class \Magento\Persistent\Test\Unit\Helper\SessionTest
+ */
+class SessionTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|Context
+     */
+    private $context;
+
+    /**
+     * @var  \PHPUnit_Framework_MockObject_MockObject|SessionHelper
+     */
+    private $helper;
+
+    /**
+     * @var  \PHPUnit_Framework_MockObject_MockObject|DataHelper
+     */
+    private $dataHelper;
+
+    /**
+     * @var  \PHPUnit_Framework_MockObject_MockObject|CheckoutSession
+     */
+    private $checkoutSession;
+
+    /**
+     * @var  \PHPUnit_Framework_MockObject_MockObject|SessionFactory
+     */
+    private $sessionFactory;
+
+    /**
+     * @var  \PHPUnit_Framework_MockObject_MockObject|Session
+     */
+    private $session;
+
+    /**
+     * Setup environment
+     */
+    protected function setUp()
+    {
+        $this->context = $this->getMockBuilder(Context::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->dataHelper = $this->getMockBuilder(DataHelper::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->checkoutSession = $this->getMockBuilder(CheckoutSession::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->scopeConfig = $this->getMockBuilder(ScopeConfigInterface::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->sessionFactory = $this->getMockBuilder(SessionFactory::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['create'])
+            ->getMock();
+        $this->session = $this->getMockBuilder(Session::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->sessionFactory->expects($this->any())->method('create')->willReturn($this->session);
+
+        $this->helper = $this->getMockBuilder(SessionHelper::class)
+            ->setMethods(['getSession'])
+            ->setConstructorArgs(
+                [
+                    'context' => $this->context,
+                    'persistentData' => $this->dataHelper,
+                    'checkoutSession' => $this->checkoutSession,
+                    'sessionFactory' => $this->sessionFactory
+                ]
+            )
+            ->getMock();
+    }
+
+    /***
```

Comment:
Please use the same style for `phpdoc` comments `/**`

Reply:
It is done. Please check it.